app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,205 @@
+    <string name="shop_name">Tên cửa hàng: </string>
+    <string name="product_quantity">Số lượng sản phẩm: </string>
+    <string name="owner">Chủ cửa hàng: </string>
+    <string name="rating">Xếp hạng: </string>
+    <string name="comments">Bình luận</string>
+    <string name="products_shop">Sản phẩm trong shop</string>
+    <string name="see_all">Xem tất cả</string>
+    <string name="title_shop">Cửa hàng :\u0020 </string>
+    <string name="shop_information">Thông tin cửa hàng</string>
+    <string name="shop_owner_information">Thông tin chủ cửa hàng</string>
+    <string name="time">Thời gian:</string>
+    <string name="pending">Chờ</string>
+    <string name="accepted">Chấp nhận</string>
+    <string name="rejected">Từ chối</string>
+    <string name="completed">Hoàn thành</string>
+    <string name="step_one_check_order">Bước 1: Kiểm tra đơn hàng</string>
+    <string name="step_two_close_order">Bước 2: Đóng đơn hàng</string>
+    <string name="continue_check">Tiếp tục</string>
+    <string name="shop_related">Cửa hàng liên quan</string>
+    <string name="add_a_comment">Thêm bình luận</string>
+    <string name="quantity">Số lượng : %d</string>
+    <string name="prices">Giá : %s</string>
+    <string name="start_date">Ngày bắt đầu </string>
+    <string name="end_date">Ngày kết thúc </string>
+    <string name="filter">Lọc</string>
+    <string name="your_order_history">Lịch sử đặt hàng của bạn</string>
+    <string name="accept">Chấp nhận</string>
+    <string name="reject">Từ chối</string>
+    <string name="select_all">Chọn tất cả</string>
+    <string name="see_more">Xem thêm..</string>
+    <string name="button_join_domain">Nút tham gia vào miền</string>
+    <string name="image_shop_management">Ảnh quản lý cửa hàng</string>
+    <string name="domain_location">Vị trí miền</string>
+    <string name="request">Yêu cầu</string>
+    <string name="cancel">Hủy</string>
+    <string name="are_you_sure_you_want_to_cancel">Bạn có chắc chắn muốn hủy?</string>
+    <string name="edit">Chỉnh sửa</string>
+    <string name="avatar_shop">Ảnh cửa hàng</string>
+    <string name="create_new_shop">Tạo cửa hàng</string>
+    <string name="open_forever">Mở cửa hằng ngày:</string>
+    <string name="auto_reject_time">Thời gian tự động từ chối: </string>
+    <string name="cover_image">Ảnh bìa</string>
+    <string name="two_dot">:</string>
+    <string name="choose_image">Chọn ảnh</string>
+    <string name="create">Tạo</string>
+    <string name="open_time">Thời gian mở cửa :</string>
+    <string name="open_daily">Mở cửa hằng ngày :</string>
+    <string name="avatar">Ảnh đại diện : </string>
+    <string name="time_automatically_refused_order">Thời gian tự động từ chối đơn đặt hàng :</string>
+    <string name="choose_file">Chọn tệp</string>
+    <string name="edit_shop">Chỉnh sửa cửa hàng</string>
+    <string name="choose_time">Chọn thời gian</string>
+    <string name="time_auto_close">Thời gian tự động đóng cửa :</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.time_auto_close` appears to be unused
 Unused resources make applications larger and slow down builds.